assets/alertmanager/secret.yaml

```diff
@@ -29,17 +29,10 @@ stringData:
       - "severity = warning"
       "target_matchers":
       - "severity = info"
-    - "equal":
-      - "namespace"
-      "source_matchers":
-      - "alertname = InfoInhibitor"
-      "target_matchers":
-      - "severity = info"
     "receivers":
     - "name": "Default"
     - "name": "Watchdog"
     - "name": "Critical"
-    - "name": "null"
```

Comment:
I'm thinking about some gitops scripts that get applied to all new clusters and that suppose the existence of the `null` receiver on openshfit..
(same thing for the `InfoInhibitor` rule, "no need to add it, openshift already does..." we don't state that we support that explicitly, but still)

Reply:
(users would get alerts in these two cases though)

Reply:
I'm not sure to follow but I expect that users managing lots of new clusters have their own default Alertmanager configuration and don't rely on the default one.

Reply:
I was thinking about users that will only modify the `route` field of this config to add routes to`null` assuming that the receiver `null` is there (but I think that alertmanager will warn about this).

I was also thinking about `AlertmanagerConfig` that would send to `null` in the default config, but I think `AlertmanagerConfig` should define their own receivers: https://github.com/prometheus-operator/prometheus-operator/blob/4cf0f5eae722add3d17530004abbef002e6dad08/pkg/alertmanager/validation/v1beta1/validation.go#L45,

Reply:
Discussed offline: we provide no stability guarantee on the default Alertmanager configuration. The assumption is that users would overwrite the out-of-the-box secret instead of patching it so altering the default config shouldn't break users.